release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -377,6 +377,23 @@ and no other Reports or ReportQueries depend on the expiring Report, the Meterin
 removes the Report from your cluster at the end of its retention period. For more information,
 see metering Reports xref:../metering/reports/metering-about-reports.adoc#metering-expiration_metering-about-reports[expiration].
 
+[id="ocp-4-6-nodes"]
+=== Nodes
+
+[id="ocp-4-6-descheduler-strategy"]
+==== New descheduler strategy is available (Technology Preview)
+
+The descheduler now allows you to configure the `PodLifeTime` strategy. This strategy evicts pods after they reach a certain, configurable age.
+
+See xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler-strategies_nodes-descheduler[Descheduler strategies] for more information.
+
+[id="ocp-4-6-descheduler-filtering"]
+==== Descheduler filtering by namespace and priority (Technology Preview)
+
+You can now configure whether descheduler strategies should consider pods for eviction based on their namespace and priority.
+
+See xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler-filtering-namespace_nodes-descheduler[Filtering pods by namespace] and xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler-filtering-priority_nodes-descheduler[Filtering pods by priority] for more information.
```

Comment:
These two links are resolving to the top of the 
Evicting Pods using the descheduler page.

Reply:
@bmcelvee Yeah sorry, I mentioned that in an earlier comment in the PR, should have mentioned it in slack too. These won't go down to the proper location until #26224 is merged.